tests/acceptance/fixtures/responses/universal.js

```diff
@@ -1,5 +1,44 @@
 import { RequestMock } from 'testcafe';
 
+function generateAutoCompleteResponse (prompt) {
+  const mockedResponse = {
+    meta: {
+      uuid: '01802d71-9901-1b83-9d50-ff143088f1ab',
+      errors: []
+    },
+    response: {
+      input: {
+        value: prompt,
+        queryIntents: []
+      },
+      results: []
+    }
+  };
+
+  if (prompt === '') {
+    mockedResponse.response.results = [
+      {
+        value: 'a Rose by any other name',
+        matchedSubstrings: [],
+        queryIntents: [],
+        verticalKeys: []
+      },
+      {
+        value: 'amani farooque phone number',
+        matchedSubstrings: [],
+        queryIntents: [],
+        verticalKeys: []
+      }
+    ];
+  } else if (prompt.includes('a')) {
+    mockedResponse.response.results = [
+      { value: 'amani farooque phone number', matchedSubstrings: [], queryIntents: [], verticalKeys: [] }
+    ];
+  }
```

Comment:
do we need to separate the autocomplete response based on prompt like this? I think the one test in acceptancesuite.js with `searchComponent.enterQuery('ama');` should be able to select the second option of the mocked results set in the if statement above with `getAutoComplete().selectOption(...)`. We may only need to return the same mocked autocomplete response for all the tests so that would simplify things here.

Reply:
Technically, the tests would pass if we just used one autocomplete response. But, that wouldn't be a realistic mocking of LiveAPI. In the test we also enter the query 'Tom'. Returning query suggestions not containing that string would be strange. I worry that if we don't realistically mock LiveAPI, that could lead us to write bad tests. As an example, someone might add some logic to our acceptance tests that looks for this query suggestion when someone enters "Tom". But, in the real world, "Tom" produces no query suggestions.

Reply:
Good call! Added the `startsWith`.